README.md

```diff
@@ -295,12 +295,11 @@ You can view [the source code](https://github.com/blockstack/blockstack-core/blo
 
 ## Understand how the shared FAQs work
 
-The FAQ system servers single-sourced content that support the FAQs that appear in blockstack.org, app.co, and stacks.com site. We have FAQs that fall into these categories:
+The FAQ system servers single-sourced content that support the FAQs that appear in blockstack.org, and stacks.com site. We have FAQs that fall into these categories:
```

Comment:
Don't think we own Stacks.com, typo from the original docs. Should probably be StacksToken.com

Reply:
Fixed